lms/static/sass/shared/_header.scss

```diff
@@ -162,7 +162,7 @@ header.global {
         position: absolute;
         top: 4px;
         @include transition(all 0.15s linear 0s);
-        width: 26px;
+        width: 19px;
```

Comment:
The image of the house here is a background image, and said image is only 19px wide. We had 26px with the image left-aligned in that "space". When viewing this in RTL it was causing a wider gap on the right (see @sarina's) screen captures above. I reduced the width of the "space" to equal the width of the house icon. This resolves the funky space issue that was mentioned above.